csrc/mmdeploy/net/rknn/CMakeLists.txt

```diff
@@ -2,19 +2,36 @@
 
 project(mmdeploy_rknn_net)
 
+mmdeploy_add_module(${PROJECT_NAME} rknn_net.cpp)
+
 add_library(rknn SHARED IMPORTED)
 
-if(DEFINED ENV{RKNPU2_DEVICE_DIR})
+if (DEFINED ENV{RKNPU2_DEVICE_DIR})
     file(TO_CMAKE_PATH $ENV{RKNPU2_DEVICE_DIR} RKNPU2_DEVICE_DIR)
-else()
-    message(FATAL_ERROR "RKNPU2_DEVICE_DIR env must be defined")
-endif()
+endif ()
+if (DEFINED RKNPU2_DEVICE_DIR)
```

Comment:
Compiling failed for RKNPU if `RKNPU2_DEVICE_DIR` is also given in env variables.